specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2017-12-01-preview/postgresql.json

```diff
@@ -1196,22 +1193,13 @@
         ],
         "responses": {
           "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/ServerAdministratorResource"
-            }
+            "description": "OK"
```

Comment:
I don't understand this change.  Is there no body on any of these responses?  Normally there is a body on 200, and there may be one on 202, depending on the API.